pkgs/applications/misc/hyper/default.nix

```diff
@@ -11,11 +11,11 @@ let
   ];
 in
 stdenv.mkDerivation rec {
-  version = "0.7.6";
-  name = "hyperterm-${version}";
+  version = "0.8.3";
+  name = "hyper-${version}";
   src = fetchurl {
```

Comment:
Oh, I've missed that package was written with dpkg in mind, sorry! If you don't want to rewrite it to compile from source, then just leave it as it was.

Also adding `hyperterm = self.hyper;` will be probably a good idea, so nothing will break for users.